setup/action.yaml

```diff
@@ -95,6 +95,20 @@ runs:
 
     - uses: suzuki-shunsuke/tfaction/export-aws-secrets-manager@main
 
+    - uses: google-github-actions/auth@55bd3a7c6e2ae7cf1877fd1ccb9d54c0503c457c # v2.1.2
+      id: google-backend-credentials
+      if: steps.target-config.outputs.gcp_remote_backend_service_account != ''
+      with:
+        create_credentials_file: true
```

Comment:
The default value is `true`, so you don't have to set this explicitly.

https://github.com/google-github-actions/auth?tab=readme-ov-file#inputs-miscellaneous